docs/user_guides/fs/feature_group/advanced_data_validation.md

```diff
@@ -0,0 +1,276 @@
+# Advanced Data Validation Options and Best Practices
+
+The introduction to data vaildation guide can be found [here](data_validation.md). The notebook example to get started with Data Validation in Hopsworks can be found [here](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/integrations/great_expectations/fraud_batch_data_validation.ipynb).
+
+## Data Validation Configuration Options in Hopsworks
+
+### Validation Ingestion Policy
+
+Depending on your use case you can setup data validation as a monitoring or gatekeeping tool when trying to insert new data in your Feature Group. Switch behaviour by using the `validation_ingestion_policy` kwarg:
+
+- `"ALWAYS"` is the default option and will attempt to insert the data regardless of the validation result. Hassle free, it is ideal to monitor data ingestion in a development setup.
+- `"STRICT"` is the best option for production ready projects. This will prevent insertion of DataFrames which do not pass all data quality requirements. Ideal to avoid "garbage-in, garbage-out" scenarios, at the price of a potential loss of data. Check out the best practice section for more on that.
+
+#### In the UI
+
+Go to the Feature Group edit page, in the Expectation section you can choose between the options above.
+
+#### In the python client
+
+```python3
+fg.expectation_suite.validation_ingestion_policy = "ALWAYS" # "STRICT"
+```
+
+If your suite is registered with Hopsworks, it will persist the change to the server.
+
+### Disable Data Validation
+
+Should you wish to do so, you can disable data validation on a punctual basis or until further notice.
```

Comment:
From what I can see after all the options to disable need you to act to enable it again. So I would remove the "on a punctual basis or until further notice". Otherwise I would expect to have some option to tell it to ignore the validation at insertion time or something like that.